template/api/lead-form-submissions/models/lead-form-submissions.settings.json

```diff
@@ -8,11 +8,12 @@
   "options": {
     "increments": true,
     "timestamps": true,
-    "draftAndPublish": true
+    "draftAndPublish": false
   },
   "attributes": {
     "email": {
-      "type": "string"
+      "type": "string",
+      "unique": true
```

Comment:
I don't think this should be unique. This would break websites that have several forms with different purposes

Reply:
Yeah good point, I used that just to throw an error, forgot to remove it.  Will do, thanks!